katalog/thanos/thanos-minio/base/minio-ha/prometheusrules.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: minio-monitoring-rules
+  namespace: monitoring
+spec:
+  groups:
+    - name: minio.rules
+      rules:
+        - alert: MinioClusterDiskOffline
```

Comment:
I would also add a name to the alert, like MonitoringMinioCluster etc

Reply:
fixed